### PR TITLE
GitHub actions setup

### LINKED ISSUE #52: GitHub Actions Backend

### LINKED ISSUE #53: Deploy Backend to cloud

### PR DESCRIPTION
**Pull request by Nephim**
Implemented GitHub action build Mobile app + Deploy to azure

### CODE REVIEW

**Review by Gormiee — APPROVED** [review]
Very cool. LGTM

File: .github/workflows/main_uicrafter.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: windows-latest
```
Comment (Aagaard00) [inline .github/workflows/main_uicrafter.yml:14]: The apk runs on windows-2022. Is that important?

**Review by Aagaard00 — APPROVED** [review]
LGTM

File: .github/workflows/main_uicrafter.yml (hunk 1 above)
Comment (Nephim) [reply .github/workflows/main_uicrafter.yml:14]: I think latest is fine but it needs to run on windows and not linux, Ill change to latest

**Pull request merged**